sql/moz-fx-data-shared-prod/braze_derived/subscriptions_map_v1/checks.sql

```diff
@@ -1,10 +0,0 @@
---macro checks
-
-#fail
-{{ not_null(["braze_subscription_name", "description", "mozilla_subscription_id", "firefox_subscription_id", "mozilla_dev_subscription_id", "basket_slug"]) }}
-
-#fail
-{{ min_row_count(1) }}
-
-#fail
-{{ is_unique(["mozilla_subscription_id", "firefox_subscription_id", "mozilla_dev_subscription_id"]) }}
```

Comment:
I believe this will check that the combination of all of those columns is unique:
https://github.com/mozilla/bigquery-etl/blob/main/tests/checks/is_unique.jinja

the `is_unique` saved metric in bigeye config would check each column individually therefore this is not exactly what you need to translate this check to bigeye metric. You will need to use this:
https://github.com/mozilla/bigquery-etl/pull/7862 (once it is merged)

Reply:
ok, in that case the bigConfig configuration should be correct.